includes/class.llms.user.permissions.php

```diff
@@ -55,19 +56,27 @@ public function __construct() {
 	 */
 	public function editable_roles( $all_roles ) {
 
+		$roles = array();
+
 		$user = wp_get_current_user();
 
 		$lms_roles = self::get_editable_roles();
 
-		foreach ( $lms_roles as $role => $allowed_roles ) {
+		foreach ( $user->roles as $user_role ) {
+			if ( 'administrator' === $user_role ) {
+				return array_keys( $all_roles );
```

Comment:
```suggestion
				return $all_roles;
```
why do return the keys of the input array?
if you visit, as administrator, the wp-admin/users.php you're flooded (depending on the number of users of course) of php notices like:
`PHP Warning:  Illegal string offset 'name' in /srv/www/wordpress-one/public_html/wp-admin/includes/template.php on line 938`
and the roles select in the user edit screen looks like:
![Schermata 2020-10-23 alle 16 44 47](https://user-images.githubusercontent.com/7689242/97018169-1e4e9c00-154f-11eb-9870-7d068d27fc4f.png)

Reply:
Ah, sorry, this was based on my previous wrong understanding that `$all_roles` came from `LLMS_Roles::get_roles()`